### PR TITLE
New package: sdcv.

### PR DESCRIPTION
**Pull request by CasperVector**
short_desc="Console version of Stardict program"

### CODE REVIEW

File: srcpkgs/sdcv/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'sdcv'
+pkgname=sdcv
+version=0.5.0beta2
```
Comment (maxice8) [inline srcpkgs/sdcv/template]: 0.5.2

File: srcpkgs/sdcv/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+revision=1
+wrksrc=$pkgname-${version/beta/-beta}-Source
+build_style=cmake
+hostmakedepends="glib-devel"
+makedepends="zlib-devel readline-devel"
+short_desc="Console version of Stardict program"
+maintainer="Casper Ti. Vector <caspervector@gmail.com>"
+license="GPL-2"
+homepage="http://sdcv.sourceforge.net"
```
Comment (maxice8) [inline srcpkgs/sdcv/template]: homepage=https://dushistov.github.io/sdcv/

File: srcpkgs/sdcv/template [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+distfiles="$SOURCEFORGE_SITE/$pkgname/$wrksrc.tar.bz2"
```
Comment (maxice8) [inline srcpkgs/sdcv/template]: distfiles="https://github.com/Dushistov/sdcv/releases"

File: srcpkgs/sdcv/template (hunk 1 above)
Comment (CasperVector) [reply srcpkgs/sdcv/template]: Many thanks; not sure how I missed the update :(

**Comment by CasperVector** [comment]
Templated updated, with pkg-config added to `$hostmakedepends`.

**Comment by CasperVector** [comment]
The problem is not about pkg-config, but about the absence of glib-devel in `$makedepends` (not `$hostmakedepends`), just like #12295; templated updated, with cross build passed on my machine.

**Pull request closed without merging**